Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:DischargeAirStaticPressureSetpointHighLimit;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Discharge Air Static Pressure Setpoint High Limit"
+  },
+  "description": {"en": "The high limit associated with discharge (supply) air static pressure"},
```

Comment:
Can we be a bit more concise in the description? In reading this, I'm not sure if it is defining a limit on the discharge air static pressure sensor or setpoint.

Reply:
Agreed, I'll make an update